index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon | Experts in SEO</title>
+    <!--Descriptive title-->
+    <link
+      rel="icon"
+      href="./assets/images/title_image.png"
+      type="image/png"
+      alt="The letter H with a blue background."
+    />
+    <!--Icon for the website-->
+  </head>
+
+  <body>
+    <header>
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!--header section with nav containing links-->
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+      <!--navigation bar with webpage links-->
+    </header>
+
+    <div class="hero"></div>
+    <!--Banner image purely decorative-->
+    <main>
+      <!--Main section containing  3 sections-->
+      <section>
```

Comment:
Instead of wrapping the div's in a section tag, you could use section tag for each `div`. The `main` tag wraps the sections within it so the `section` is redundant in this case. 

```for example
   <main>
      <section> 
          Section 1 
     </section>
     <section> 
          Section 2
     </section>
  </main>
```

Apply this throughout.